deployment/postgres-init-db/sql/ui/tables/extension_roles.sql

```diff
@@ -0,0 +1,18 @@
+create table extension_roles (
+  extension_id  integer not null,
+  id integer generated always as identity,
+  role text not null,
+
+  constraint extension_roles_primary_key primary key (id),
+  constraint extension_roles_to_extension
+    foreign key (extension_id)
+      references "extensions"
+      on delete cascade
```

Comment:
Minor: consistency: even FKs against generated PKs, we add `on update cascade`
```suggestion
    foreign key (extension_id)
      references "extensions"
      on update cascade
      on delete cascade
```